### DISCUSSION TITLE
GKD网页审查工具增强油猴脚本

### DISCUSSION THREAD

**Discussion by adproqwq** [gkd-kit/gkd#719] [General]
GresyFork：https://greasyfork.org/zh-CN/scripts/488829
ScriptCat：https://scriptcat.org/zh-CN/script-show-page/2322/
Github：https://github.com/adproqwq/MikuMagicWand

内含复制增强、快捷搜索指定选择器、自动设定showSize、节点隐私打码

**Comment by AIsouler** [comment]
好用！
